### PR TITLE
DLNA subtitles support

### LINKED ISSUE #18: DLNA Subtitles
I'd need to add a path to the server created [here](https://github.com/feross/webtorrent/blob/master/lib/server.js) to serve the subtitles via DLNA. 

At the moment the server only admits requests to `/` and `/[number]` and obviously this would be needed in rare occasions so maybe we could add a method to add a path or something like that?

### PR DESCRIPTION
**Pull request by iovis9**
Fixes #18

### CODE REVIEW

File: bin/cmd.js [hunk 1]
```diff
@@ -98,6 +100,13 @@ if (argv.subtitles) {
   MPLAYER_EXEC += ' -sub ' + JSON.stringify(argv.subtitles)
   MPV_EXEC += ' --sub-file=' + JSON.stringify(argv.subtitles)
   OMX_EXEC += ' --subtitles ' + JSON.stringify(argv.subtitles)
+
+  var subtitlesServer = http.createServer(
```
Comment (feross) [inline bin/cmd.js]: What if there is no `argv.subtitles`? Can you add an if-statement here?

**Comment by feross** [comment]
LGTM, I'll make the change I commented about before releasing a new version.

File: bin/cmd.js (hunk 1 above)
Comment (feross) [inline bin/cmd.js]: Oh, nevermind. Just saw that it's nested in an if-statement already. I misread the diff.

**Comment by feross** [comment]
I made a change to reduce the repeated arguments () and published this as 1.5.0. @iovis9 Can you test it out and confirm that everything still works for you?

**Comment by iovis9** [comment]
Still working and a lot cleaner!

**Pull request merged**